Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1635,6 +1644,8 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECOPROD1']=merge([{ '-s' : 'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT', '--datatier' : 'AODSIM,MINIAODSIM', '--eventcontent' : 'AODSIM,MINIAODSIM'},step3Defaults])
 #steps['RECOPRODUP15']=merge([{ '-s':'RAW2DIGI,L1Reco,RECO,EI,DQM:DQMOfflinePOGMC','--datatier':'AODSIM,DQMIO','--eventcontent':'AODSIM,DQM'},step3Up2015Defaults])
 steps['RECOPRODUP15']=merge([{ '-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT,DQM:DQMOfflinePOGMC','--datatier':'AODSIM,MINIAODSIM,DQMIO','--eventcontent':'AODSIM,MINIAODSIM,DQM'},step3Up2015Defaults])
+## for 2017 PROD
+steps['RECOPRODUP17']=merge([{ '--era' :'Run2_2017','-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT','--conditions': 'auto:phase1_2017_realistic'},steps['RECOPRODUP15']])
```

Comment:
Since you are not running DQM step, you have to remove DQM/DQMIO from eventcontent/datatier